tools/kafka/docker-compose/mysql.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
```

Comment:
These are slated to go in under `tools/kafka`. I wonder if we should instead add a `tools/mysql` and `tools/postgres`? Or maybe this is instead under `tools/scs-schema-registry`. Wdyt?

Reply:
Good point. I think we can move them to the root of this set of samples (`scst-schema-registry-integration`). They are only applicable to these samples and we don't use them anywhere else.